### PR TITLE
When we connect with a tool, we don't want to see error messages about

### PR DESCRIPTION
**Pull request by dsolt**
each connection method it is trying.

### CODE REVIEW

**Comment by ibm-ompi** [comment]
The IBM CI (PGI Compiler) build failed! Please review the log, linked below.

 Gist: https://gist.github.com/ibm-ompi/88b480fb98dfd2bdac939f023e1f8fe6

**Comment by rhc54** [comment]
@dsolt Just a couple of points for the future:

1. please don't create these as branches on the community repo. Fork the repo into your personal github area, create the branch on the fork, and then generate the PR back to the community repo. You don't have to redo these you filed today, but let's use the normal practice going forward.

1. we generally require changes to go into master and then be cherry-picked back to the release branches. In some of these cases, I suspect there are no equivalents on master. I'll check and deal with these - again, we should just watch for it going forward.

File: src/mca/ptl/tcp/ptl_tcp.c [hunk 1]
```diff
@@ -535,7 +536,9 @@ static pmix_status_t try_connect(int *sd)
 
     /* establish the connection */
     if (PMIX_SUCCESS != (rc = pmix_ptl_base_connect(&mca_ptl_tcp_component.connection, len, sd))) {
-        PMIX_ERROR_LOG(rc);
+    /* Do not be noisy about this.  It is normal to try and fail different
+       connection methods until we find the right one */
+//        PMIX_ERROR_LOG(rc);
```
Comment (rhc54) [inline src/mca/ptl/tcp/ptl_tcp.c]: I'd prefer to just remove this line, but can live with it this way

**Comment by jjhursey** [comment]
bot:ibm:pgi:retest

**Pull request merged**